xformers/triton/softmax.py

```diff
@@ -200,7 +200,7 @@ def _softmax_dispatch(
         _triton_registered_warnings = True
 
     if mask is not None:
-        x += mask
+        x = x + mask
```

Comment:
@blefaudeux how come this change?

Reply:
it changes the operation from inplace to not-in place, looks like nothing but this was breaking the backward pass (and was not tested before, so this flew under the radar)